Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1809,11 +1809,13 @@ namespace Remotion.BocAutoCompleteReferenceValue
             maxHeight = '';
         }
 
-        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference);
+        const bufferFromLeft = 30;
+        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference) - bufferFromLeft;
```

Comment:
If the input element is less than 30px away from the left side we still use the 30px as spacing. We should only reduce the left space if there is any. 
```suggestion
        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference) - Math.min(bufferFromLeft, position.left);
```